website/docusaurus.config.js

```diff
@@ -124,9 +138,9 @@ module.exports = {
       {
         docs: {
           sidebarPath: require.resolve('./sidebars.js'),
-          editUrl:
-            'https://github.com/TypeStrong/ts-node/edit/docs/website/',
+          editUrl: 'https://github.com/TypeStrong/ts-node/edit/docs/website/',
         },
+        blog: false,
```

Comment:
`blog: undefined` would still initialize the blog plugin, which would create some default routes like `/blog/archive`, even if there aren't any posts.